### PR TITLE
Standalone SELFBALANCE opcode

### PR DESCRIPTION
**Pull request by bobsummerwill**

### CODE REVIEW

File: _specs/ecip-standalone-self-balance.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+lang: en
+ecip: Unassigned
```
Comment (sorpaas) [inline _specs/ecip-standalone-self-balance.md]: Assign number ECIP-1080.

**Comment by bobsummerwill** [comment]
ECIP Editors: @soc1c @YazzyYaz @meowsbits @sorpaas @BelfordZ:
Please can you review/comment/merge?  I think this is ready to go.  Thanks!

File: _specs/ecip-1080.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The following clients with Ethereum Classic support already implement the SELFBALANCE opcode, but
+not in this standalone form yet.  The work to support standalone would be minimal.
+
+- Parity Ethereum
+- Multi-Geth
+- Hyperledger Besu
+
+
+## Copyright
```
Comment (sorpaas) [inline _specs/ecip-1080.md]: To comply with the Apache-2.0 license, please include the content of the NOTICE file in this section: https://corepaper.org/NOTICE

Comment (sorpaas) [inline _specs/ecip-1080.md]: To comply with the Apache-2.0 license, please include the content of the NOTICE file here: https://corepaper.org/NOTICE

**Comment by bobsummerwill** [comment]
@sorpaas Is this blocked from merging purely on your review?  Please can you unblock and merge please?  Thank you.

**Comment by bobsummerwill** [comment]
@sorpaas Copyright language corrected.  This good now?

**Comment by BelfordZ** [comment]
solid op code

**Pull request merged**